site2/docs/reference-cli-bookkeeper.md

```diff
@@ -1,10 +1,5 @@
----
-id: reference-cli-bookkeeper
-title: Pulsar CLI tools - bookkeeper
-sidebar_label: "cli-bookkeeper"
----
```

Comment:
Why do you make these chanes? If you'd like to set up redirections, I suggest:

```
````mdx-code-block
import {Redirect} from '@docusaurus/router';
<Redirect to="/reference" />
````
```

Reply:
@SignorMercurio can I use this? Does it work?

Reply:
No, the aim is not redirecting. Actually, `reference-cli-bookkeeper.md` (and the below) will not be shown on `pulsar.apache.org`. Rather, it'll be shown only in `pulsar.apache.org/reference`. Therefore, the frontmatter is no longer useful as it only works for Docusaurus.